### DISCUSSION TITLE
Gameplay does not pause immediately after resuming

### DISCUSSION THREAD

**Discussion by luaneko** [ppy/osu#13051] [Ideas]
**Describe the bug:** Gameplay does not pause immediately after resuming.

Not sure if this design was intentional, but the way it's currently implemented is definitely annoying.

Steps to reproduce:

1. Pause gameplay using escape
2. Resume
3. Immediately try pausing again using escape. The circular exit indicator appears but nothing happens. The game does not register escape keys for another one or two seconds at all.

**Screenshots or videos showing encountered issue:** 
https://streamable.com/m7rhc
https://streamable.com/zfntd

**osu!lazer version:** 2019.830.0

**Logs:** irrelevant

**Comment by smoogipoo** [comment]
Putting this as a proposal for now, as it's intended but maybe needs a different display.

**Comment by luaneko** [comment]
Being able to see how many seconds are left until I can pause again would be great.

**Comment by frenzibyte** [comment]
Maybe flashing red color that will fade out with duration of the timeout?

**Comment by peppy** [comment]
Showing a countdown in the centre of the hold to pause button is best.
